modules/social_features/social_group/src/Controller/DeleteGroup.php

```diff
@@ -47,12 +47,12 @@ public static function deleteGroupAndContentFinishedCallback($success, $results,
         'One item deleted.', '@count items deleted.'
       );
       // Provide some feedback when its a success.
-      drupal_set_message(t('Your group and all of its topics, events and posts have been deleted.'));
-      // TODO: log to the database.
+      \Drupal::messenger()->addStatus(t('Your group and all of its topics, events and posts have been deleted.'));
```

Comment:
Ideally this should come via Dependeny Injection.

Reply:
Ideally yes, but this is necessary for D9 compatibility. We have accepted Dependency injection not to be fixed.